pkg/cidata/user-data.TEMPLATE

```diff
@@ -91,6 +219,15 @@ write_files:
         {{- if .Containerd.User}}
         dnf install -y shadow-utils
         {{- end}}
+      elif command -v apk 2>&1 >/dev/null; then
+        : {{/* make sure the "elif" block is never empty */}}
+        {{- if .Mounts}}
+        if ! command -v sshfs 2>&1 >/dev/null; then
+          apk update
+          apk add sshfs
```

Comment:
apk update is probably not needed

Reply:
It is needed, at least in my image, which only includes a static apkovl in the repositories by default.

It is still 🚧, but the script I use to build (various different) ISOs is at https://github.com/rancher-sandbox/alpine-lima

Reply:
When I tested it again right now, it seemed like `apk update` is indeed not needed. Not sure which image I was testing when I did need it.  I can remove it, if you prefer, or leave it in. It doesn't really hurt, as `apk add sshfs` will still update the repos anyways, so I would prefer to leave it as-is.